### DISCUSSION TITLE
是否可以增加一个去掉分区栏的外观按钮，我感觉这样更好看呢！

### DISCUSSION THREAD

**Discussion by Jyuan-Ip** [magicdawn/Bilibili-Gate#62] [Ideas]
我这个是结合了另一个油猴插件实现的效果，但是我觉得我们这个插件其实是可以增加这个功能的！！！
![{C3215FAC-E217-4923-AAC3-D37FE2FE9C8E}](https://github.com/magicdawn/bilibili-app-recommend/assets/87629069/2cc9d482-5521-4e46-800b-a31d937b371e)

**Comment by Jyuan-Ip** [comment]
还有！！我觉得行与行之间的距离其实可以再缩小一点点的，目前看起来其实是有一点点宽的😊😊💕

**Reply by magicdawn** [reply]
现在有头像的这样样式固定 40px, 下个版本会对小屏处理一下. 
可以使用 CI build 体验 https://github.com/magicdawn/bilibili-app-recommend/raw/release-nightly/bilibili-app-recommend.mini.user.js

**Comment by magicdawn** [comment]
你可以将你使用的另一个脚本贴在这里, 这样有同样需求的人就可以直接用了 🐶

**Reply by Jyuan-Ip** [reply]
应该是这个😁
https://github.com/the1812/Bilibili-Evolved

**Reply by Jyuan-Ip** [reply]
但是还是希望作者大大可以内置这个功能，不是很想为了这个效果而多开启另一个插件💕😊

**Reply by magicdawn** [reply]
Bilibili-Evolved 已有的功能不会再内置, 我自己也是同时在用 Bilibili-Evolved(自定义顶栏 和 夜间模式)

**Reply by Jyuan-Ip** [reply]
好的！！！

**Comment by magicdawn** [comment]
已内置~
